.circleci/config.yml

```diff
@@ -3,15 +3,25 @@ version: 2.1
 orbs:
   buildevents: honeycombio/buildevents@0.2.3
 
+matrix_goversions: &matrix_goversions
+  matrix:
+    parameters:
+      goversion: ["12", "13", "14", "15", "16", "17"]
+
+# Default version of Go to use for Go steps
+default_goversion: &default_goversion "17"
+
 executors:
   go:
     parameters:
       goversion:
         type: string
-        default: "12"
-    working_directory: /go/src/github.com/honeycombio/dynsampler-go
+        default: *default_goversion
+    working_directory: /home/circleci/go/src/github.com/honeycombio/dynsampler-go
```

Comment:
I don't _think_ working directory is necessary. I removed it recently on another repo PR that updated the docker image

Reply:
It was present here and in libhoney-go. But let's find out! 🧑‍🔬  :atom:  ⚗️ 🧪

Reply:
This was the PR: https://github.com/honeycombio/buildevents/pull/132/files

Reply:
![Nice!](https://user-images.githubusercontent.com/517302/148616436-98c3bcda-10bd-433d-85ff-fcf6d846755c.gif)